picard/coverart/utils.py

```diff
@@ -41,8 +41,10 @@
 
 
 def translate_caa_type(name):
-    if name == 'unknown':
-        return _(CAA_TYPES_TR[name])
+    print("translate_caa_type", name, CAA_TYPES_TR.get(name))
+    name_lower = name.lower()
```

Comment:
This shouldn't be needed...

https://github.com/metabrainz/picard/commit/259e176144c10fd82e662591849ec6ba698ad4b1#diff-4246654415c5ccfdafff4d7925531509R360

I think this last change was incorrect, it should be:

```python
title = translate_caa_type(name)
```

`translate_caa_type()` takes the name of the CAA type, especially to avoid this. But I guess we overlooked the change in rdswift's patch.

`CAA_TYPES_TR` is a dict matching CAA name (always in English, and lower-case) to the English CAA title.

Reply:
Indeed, you are right. It's the call being wrong. I updated the PR.